### DISCUSSION TITLE
Add Support for Windows OS

### DISCUSSION THREAD

**Discussion by foodmaher** [wwmm/easyeffects#2718] [General]
Hi, everyone,

Why not provide installation exe for Windows? That will be the best equalizer on Windows even now; while I hope to use the easy effect on my laptop, my computer does not work correctly with Linux : (

Thanks .. . .

**Comment by wwmm** [comment]
There are some reasons. The first is that the last Windows I used was `Windows 98` almost 20 years ago. So I can't develop software for Windows. I do not use the system. The second is that our whole code is based on PipeWire and some other libraries that are not developed for Windows. It may not even be possible to create a Windows binary.
